Slack tests are failing

**TestSlackEvent/signed_event_ok - github.com/triggermesh/knative-sources/pkg/adapter/slacksource**
```
    Failed
    === RUN   TestSlackEvent/signed_event_ok
        --- FAIL: TestSlackEvent/signed_event_ok (0.00s)
    panic: runtime error: invalid memory address or nil pointer dereference [recovered]
    	panic: runtime error: invalid memory address or nil pointer dereference
    [signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x226da9b]

    goroutine 98 [running]:
    testing.tRunner.func1.1(0x2425740, 0x3bc8530)
    	/usr/local/go/src/testing/testing.go:988 +0x452
    testing.tRunner.func1(0xc00067ab40)
    	/usr/local/go/src/testing/testing.go:991 +0x600
    panic(0x2425740, 0x3bc8530)
    	/usr/local/go/src/runtime/panic.go:975 +0x3e3
    github.com/triggermesh/knative-sources/pkg/adapter/slacksource.(*slackEventAPIHandler).handleError(0xc0005b3320, 0x0, 0x0, 0x1f4, 0x2bdd8c0, 0xc0005a1880)
    	/home/circleci/project/pkg/adapter/slacksource/slack.go:162 +0x23b
    github.com/triggermesh/knative-sources/pkg/adapter/slacksource.(*slackEventAPIHandler).handleCallback(0xc0005b3320, 0xc00064ac80, 0x2bdd8c0, 0xc0005a1880)
    	/home/circleci/project/pkg/adapter/slacksource/slack.go:200 +0x304
    github.com/triggermesh/knative-sources/pkg/adapter/slacksource.(*slackEventAPIHandler).handleAll(0xc0005b3320, 0x2bdd8c0, 0xc0005a1880, 0xc000652900)
    	/home/circleci/project/pkg/adapter/slacksource/slack.go:136 +0x5ba
    net/http.HandlerFunc.ServeHTTP(...)
    	/usr/local/go/src/net/http/server.go:2041
    github.com/triggermesh/knative-sources/pkg/adapter/slacksource.TestSlackEvent.func1(0xc00067ab40)
    	/home/circleci/project/pkg/adapter/slacksource/slack_test.go:270 +0x855
    testing.tRunner(0xc00067ab40, 0xc00000e8c0)
    	/usr/local/go/src/testing/testing.go:1039 +0x1ec
    created by testing.(*T).Run
    	/usr/local/go/src/testing/testing.go:1090 +0x701
```

**TestSlackEvent - github.com/triggermesh/knative-sources/pkg/adapter/slacksource**
```
    Failed
    === RUN   TestSlackEvent
    === CONT  TestSlackEvent
        logger.go:130: 2020-08-18T21:53:54.977Z	ERROR	An error ocurred{error 26 0  request without body not supported}
    === CONT  TestSlackEvent
        logger.go:130: 2020-08-18T21:53:54.978Z	ERROR	An error ocurred{error 26 0  could not unmarshall JSON request: invalid character 'h' in literal true (expecting 'r')}
    === CONT  TestSlackEvent
        logger.go:130: 2020-08-18T21:53:54.979Z	WARN	not supported content ""
    === CONT  TestSlackEvent
        logger.go:130: 2020-08-18T21:53:54.980Z	ERROR	An error ocurred{error 26 0  empty signature timestamp header}
    === CONT  TestSlackEvent
        logger.go:130: 2020-08-18T21:53:54.981Z	ERROR	An error ocurred{error 26 0  error parsing header timestamp: strconv.ParseInt: parsing "abcde": invalid syntax}
    === CONT  TestSlackEvent
        logger.go:130: 2020-08-18T21:53:54.982Z	ERROR	An error ocurred{error 26 0  signing timestamp expired}
    === CONT  TestSlackEvent
        logger.go:130: 2020-08-18T21:53:54.984Z	INFO	callback received
        logger.go:130: 2020-08-18T21:53:54.985Z	ERROR	An error ocurred{ 27 0  <nil>}
    --- FAIL: TestSlackEvent (0.01s)
```

--- a/pkg/adapter/slacksource/slack.go
+++ b/pkg/adapter/slacksource/slack.go
@@ -31,6 +31,8 @@ import (
 	"go.uber.org/zap"
 )
 
+const apiAppIdCeExtension = "comslackapiappid"
+
 // SlackEventAPIHandler listen for Slack API Events
 type SlackEventAPIHandler interface {
 	Start(ctx context.Context) error
@@ -207,7 +209,7 @@ func cloudEventFromEventWrapper(wrapper *SlackEventWrapper) (*cloudevents.Event,
 	event.SetID(wrapper.EventID)
 	event.SetType(v1alpha1.SlackGenericEventType)
 	event.SetSource(wrapper.TeamID)
-	event.SetExtension("api_app_id", wrapper.APIAppID)
+	event.SetExtension(apiAppIdCeExtension, wrapper.APIAppID)
 	event.SetTime(time.Unix(int64(wrapper.EventTime), 0))
 	event.SetSubject(wrapper.Event.Type())
 	if err := event.SetData(cloudevents.ApplicationJSON, wrapper.Event); err != nil {

--- a/pkg/adapter/slacksource/slack_test.go
+++ b/pkg/adapter/slacksource/slack_test.go
@@ -14,10 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// import (
-// 	cloudevents "github.com/cloudevents/sdk-go/v2"
-// )
-
 package slacksource
 
 import (

